### PR TITLE
[Merged by Bors] - fix: minor api proxy setup (PL-927)

### PR DESCRIPTION
**Pull request by DecathectZero**
minor env var and setup for the feature

### CODE REVIEW

**Comment by DecathectZero** [comment]
* **#774** <a href="https://app.graphite.dev/github/pr/voiceflow/general-runtime/774?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#773** <a href="https://app.graphite.dev/github/pr/voiceflow/general-runtime/773?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#772** <a href="https://app.graphite.dev/github/pr/voiceflow/general-runtime/772?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @DecathectZero and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: runtime/lib/HTTPClient/function-lambda/function-lambda-client.ts [hunk 1]
```diff
@@ -136,7 +136,7 @@ export class FunctionLambdaClient {
     }
   }
 
-  private async invokeLambda(request: FunctionLambdaRequest) {
+  public async invokeLambda(request: FunctionLambdaRequest) {
```
Comment (DecathectZero) [inline runtime/lib/HTTPClient/function-lambda/function-lambda-client.ts:139]: this needs to be public to force a direct call to the lambda and bypass specific wrappers for functions.

**Review by JoshTheDeveloper — APPROVED** [review]

**Review by trs — APPROVED** [review]

**Pull request closed without merging**